api/src/main/java/org/pmiops/workbench/api/InstitutionController.java

```diff
@@ -34,6 +35,7 @@
   }
 
   @Override
+  @AuthorityRequired({Authority.INSTITUTION_ADMIN})
```

Comment:
Regular users should not be able to see the Inst's list of approved individual emails